kANe/v5p1/1_Vratas_and_Utsavas/10_vijayAdashamI_and_divALLii.md

```diff
@@ -51,240 +51,144 @@ In the case of the king the procedure described by Hemādri (on vrata vol. I. pp
 
-modern times only three out of these, viz.. worship of Bali, illu minations and gambling are observed and the rest are hardly observed anywhere. Therefore only brief references will be made to the rest. Cows, calves and bulls are to be worshipped with two mantras (quoted in note ) and decorated and cows are not to be milked and bulls are not to be made to carry burdens this day. This is to be done in the morning of Kārtika pratipad. If it be mixed up with 526 the 2nd it should not be done on that day as that would lead to the death of one's sons, wife and loss of wealth, but this should be observed when pratipad is mixed with amāyāsyā. Devala prescribes this and remarks that by the wor ship of cows three prosper, viz. people, cows and the king. As regards Govardhana-pūjā, those who are in the vicinity of Govar
+modern times only three out of these, viz.. worship of Bali, illu minations and gambling are observed and the rest are hardly observed anywhere. Therefore only brief references will be made to the rest. Cows, calves and bulls are to be worshipped with two mantras (quoted in note ) and decorated and cows are not to be milked and bulls are not to be made to carry burdens this day. This is to be done in the morning of Kārtika pratipad. If it be mixed up with [^526] the 2nd it should not be done on that day as that would lead to the death of one's sons, wife and loss of wealth, but this should be observed when pratipad is mixed with amāyāsyā. Devala prescribes this and remarks that by the wor ship of cows three prosper, viz. people, cows and the king. As regards Govardhana-pūjā, those who are in the vicinity of Govar
 
-524. बलिराज्ये दीपदानात्सदा लक्ष्मीः स्थिरा भवेत्। दीपै राजनादत्र सैषा दीपावली स्मृता । बलिराज्य समासाथ यैर्न दीपावली कृता। तेषां गृहे कथं दीपा प्रज्वलिष्यन्ति केशव ॥ धर्मसिन्धु p. 106. पु. चि. pp. 243-244. बलिराज्य extends over three days from the 14th. पुरा वामनरूपेण प्रार्थयित्वा धरामिमाम्। ददावतिथिरिन्द्राय बलिं पातालवासिनम् । कृत्वा दैत्यपतेर्दत्तमहोरात्रत्रयं तृप। १. by का. त. वि. p. 315. The printed भविष्योत्तर (140. 1-2) reads 'इन्द्राय दत्तवान् राज्य "दैत्यपतेर्वासमहोरात्रं पुनर्नृप ।'
+[^524]. बलिराज्ये दीपदानात्सदा लक्ष्मीः स्थिरा भवेत्। दीपै राजनादत्र सैषा दीपावली स्मृता । बलिराज्य समासाथ यैर्न दीपावली कृता। तेषां गृहे कथं दीपा प्रज्वलिष्यन्ति केशव ॥ धर्मसिन्धु p. 106. पु. चि. pp. 243-244. बलिराज्य extends over three days from the 14th. पुरा वामनरूपेण प्रार्थयित्वा धरामिमाम्। ददावतिथिरिन्द्राय बलिं पातालवासिनम् । कृत्वा दैत्यपतेर्दत्तमहोरात्रत्रयं तृप। १. by का. त. वि. p. 315. The printed भविष्योत्तर (140. 1-2) reads 'इन्द्राय दत्तवान् राज्य "दैत्यपतेर्वासमहोरात्रं पुनर्नृप ।'
```

Comment:
This footnote definition style is incorrect. 

It should be : `[^524]:`

Below too, the dashes before  `___[^525]` should go. Otherwise, proper footnotes are not generated. 

Further, footnotes must be moved to the end of the paragraph and not in the middle of a sentence. In this case, "As regards Govardhana-pūjā, those who are in the vicinity of Govar" can be mobed to the line starting with "dhana (a hill near Mathurā) should go to it"

Can you please correct it here as well as in past files you worked on?

Reply:
I had not touched this paragraph yesterday. I had stopped with footnote 523. 
Today I have started with 524. So, you can see the necessary corrections today.